frontend/components/queries/QueryResultsTable/QueryResultsTable.jsx

```diff
@@ -195,20 +197,26 @@ class QueryResultsTable extends Component {
               queryTimerMilliseconds={queryTimerMilliseconds}
             />
           )}
-          <Button
-            className={toggleFullScreenBtnClass}
-            onClick={onToggleQueryFullScreen}
-            variant="grey"
-          >
-            <KolideIcon name={isQueryFullScreen ? "windowed" : "fullscreen"} />
-          </Button>
-          <Button
-            className={`${baseClass}__export-btn`}
-            onClick={onExportQueryResults}
-            variant="inverse"
-          >
-            Export results
-          </Button>
+          {!hasNoResults && (
+            <>
+              <Button
+                className={toggleFullScreenBtnClass}
+                onClick={onToggleQueryFullScreen}
+                variant="grey"
+              >
+                <KolideIcon
+                  name={isQueryFullScreen ? "windowed" : "fullscreen"}
+                />
+              </Button>
+              <Button
+                className={`${baseClass}__export-btn`}
+                onClick={onExportQueryResults}
+                variant="inverse"
+              >
+                Export results
+              </Button>
+            </>
+          )}
```

Comment:
```suggestion
          <Button
            className={toggleFullScreenBtnClass}
            onClick={onToggleQueryFullScreen}
            variant="grey"
          >
            <KolideIcon
              name={isQueryFullScreen ? "windowed" : "fullscreen"}
            />
          </Button>
          {!hasNoResults && !queryIsRunning && (
          <Button
            className={`${baseClass}__export-btn`}
            onClick={onExportQueryResults}
            variant="inverse"
          >
            Export results
          </Button>
         )}
```
Looking good! I have a suggested change to render the "Toggle fullscreen" always and only renders the "Export Results" button if there are results and if the query is not running.

Loom demo to demonstrate suggestions: https://www.loom.com/share/592702dcf5b14ce7b973361cb64d7e73

Reply:
Cool! Waiting on @zwass 's input and I can go ahead and commit @noahtalerman 's suggestion.

Reply:
Sounds good to me. Otherwise I think this is good to merge!